app/Services/AI/Providers/OpenAIResponsesProvider.php

```diff
@@ -413,7 +413,7 @@ public function formatStreamChunk(string $chunk): array
             'isDone' => $isDone,
             'usage' => $usage,
             'auxiliaries' => [],
-            'skip' => empty($content) && empty($imageData) && !$isDone && !$isFirstUpdate && empty($thinking_updates),
+            'skip' => empty($content) && $content != "0" && empty($imageData) && !$isDone && !$isFirstUpdate && empty($thinking_updates),
```

Comment:
the [empty](https://www.php.net/manual/en/function.empty.php) function seems to interpret what empty means quite broadly. Is `content` always a String? Maybe this would be more robust:

```suggestion
            'skip' => trim($content) === "" && empty($imageData) && !$isDone && !$isFirstUpdate && empty($thinking_updates),
```

note: I know nothing about PHP.

Reply:
Yes, that might indeed be more stable. And $content is always a string